lib/tessel/deploy.js

```diff
@@ -225,8 +225,11 @@ actions.runScript = function(t, filepath, entryPoint) {
           resolve();
         }
 
-        // Once we get a SIGINT from the console, stop the script
-        process.once('SIGINT', scriptStopped);
+        // Once we get a SIGINT from the console
+        process.once('SIGINT', function() {
+          // pass the SIGINT to the remote process
+          remoteProcess.stdout.signal('SIGINT');
+        });
```

Comment:
When the `SIGINT` signal is sent to the remoteProcess, the remoteProcess then sends the CLI the signal that it has closed so the `remoteProcess.once('close')` event can fire (which executes the `scriptStopped` function). 

To your point, the log should probably be `Script stopped` rather than `stopping`.